dev-db/mariadb/additional-files/setup-mariadb.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# Create the mariadb system tables 
+/system/apps/MariaDB/scripts/mariadb-install-db --user=user --datadir=/system/non-packaged/mariadb/data/ --auth-root-authentication-method=normal
```

Comment:
That datadir path is wrong. `/system/non-packaged` is a "prefix" path and you shouldn't create a `mariadb` directory directly under it. I think that a path like `/system/var/mariadb/data` would be better. I see you are using a `var` path in the user hierarchy in the recipe already, so that would fit well.

For reference, this is the list of top-level directories under $prefix that haikuporter allows: https://github.com/haikuports/haikuporter/blob/49672cffc9e0f7d440ef05d68584d94a831ed581/HaikuPorter/Policy.py#L17-L36

I would keep the contents of non-packaged also like that (besides non-packaged itself of course).

Reply:
It was a bad copy paste regarding this folder. This has been adjusted to /boot/home/config/var/mariadb/data/